tests/test_Vitovent.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from PyViCare.PyViCareVentilationDevice import VentilationDevice
+from tests.ViCareServiceMock import ViCareServiceMock
+
+
+class Vitovent(unittest.TestCase):
+    def setUp(self):
+        self.service = ViCareServiceMock('response/Vitovent.json')
+        self.device = VentilationDevice(self.service)
+
+    def test_isHeatingDevice(self):
+        self.assertFalse(self.device.isHeatingDevice())
+
+    def test_isDomesticHotWaterDevice(self):
+        self.assertTrue(self.device.isDomesticHotWaterDevice())
```

Comment:
For some reason the vitovent says it is a dhw device. Maybe this is all not that accurate.

Reply:
Idk. Maybe the description is inacuarate/misleading as my vivovent has an active Heat exchanger and is capable to heat up the incoming air...

Reply:
Also it has a bypass function to bypass the heat exchanger. Maybe this what the flag is for, bypass active or inactive. This should be in the Response from the API, as the viessmann App also shows this.